index.html

```diff
@@ -0,0 +1,117 @@
+        <div class="d-flex flex-row flex-md-column card-1 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_01.png">
+          <h4 class="card-title orange-text">Lecture</h4>
+          <p class="card-description">Listen to the speakers from various countries about the messages of sharing and openign.</p>
+        </div>
+        <div class="d-flex flex-row flex-md-column card-2 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_02.png">
+          <h4 class="card-title orange-text">Exhibition</h4>
+          <p class="card-description">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+        </div>
+        <div class="d-flex flex-row flex-md-column card-3 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_03.png">
+          <h4 class="card-title orange-text">Forum</h4>
+          <p class="card-description">Have the time to share your thoughts and opinions with experts for each topic in the session.</p>
+        </div>
+        <div class="d-flex flex-row flex-md-column card-4 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_04.png">
+          <h4 class="card-title orange-text">Workshop</h4>
+          <p class="card-description">Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <div class="d-flex flex-row flex-md-column card-5 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_05.png">
+          <h4 class="card-title orange-text">Ignite</h4>
+          <p class="card-description">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+        </div>
+      </section>
+      <a class="see-program" href="">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="partners d-none d-md-flex flex-md-column" id="partners">
+      <h3>Partners</h3>
+      <div class="partners-list">
+        <p>Google</p>
+        <p>EthioTelecom</p>
+        <p>INSA</p>
+        <p>WebSprix</p>
+        <p>GDG</p>
+      </div>
+    </section>
+  </main>
+  <footer class="footer col-9 d-none d-md-flex flex-md-row">
+    <img src="img/gdg-addis-logo.svg">
+    <p>2023 GDG-Addis. Some Rights Reserved <br> <br> Google Developers Group Addis (GDG Addis) is an open community of passionate software developers 
+      and tech-savvy people in Ethiopia. We are a passionate tech-community to gather, learn, share experiences, 
+      build software technologies, and create the technology eco-system in Ethiopia. It is free and open to anyone
+      who is interested in technology. </p>
+  </footer>
```

Comment:
- [ ] **Home Page Desktop**: I would advise you to
- Redesign the pictures of each article
- The logo used in the footer must be the same logo used at the top of the page
- [ ] Your Design

![Capture](https://user-images.githubusercontent.com/63972670/227660238-74ed0a14-30c0-4fa8-8cae-e3b458b05e5d.PNG)


- [ ] Original

![Capture1](https://user-images.githubusercontent.com/63972670/227660287-eb4c05eb-ff9c-44b1-bcac-d76d84665643.PNG)